### PR TITLE
React component library

### PR DESCRIPTION
**Pull request by jeysal**
and usage in the frontend

### CODE REVIEW

**Comment by jeysal** [comment]
Oops, just noticed the build on this didn't pass :raised_eyebrow: I'll look into it

**Comment by jeysal** [comment]
https://github.com/yldio/asap-hub/pull/23 will help make the build deterministic again, then it should pass on CI as well as any local environment.

**Comment by jeysal** [comment]
rebased, should work now :pray:

File: packages/react-context/tsconfig.json [hunk 1]
```diff
@@ -4,5 +4,6 @@
     "rootDir": "src",
     "outDir": "build"
   },
-  "include": ["src"]
+  "include": ["src"],
+  "references": [{ "path": "../auth" }]
```
Comment (fampinheiro) [inline packages/react-context/tsconfig.json:8]: Can we infer this from dependencies?

Comment (jeysal) [reply packages/react-context/tsconfig.json:8]: #23 actually kind of implemented that, but only checks that they are there. TypeScript inferring them automatically is unfortunately still not possible https://github.com/microsoft/TypeScript/issues/25376

**Pull request merged**